assets/src/dashboard/templates/raw/fitness.json

```diff
@@ -2492,7 +2420,7 @@
           "type": "text",
           "basedOn": "0884aabc-ca1e-4b37-a87b-68a6fbf377db",
           "id": "96fea994-b71e-442f-b611-16d73f520016",
-          "content": "<span style=\"font-weight: 600; color: #fff\">SWEAT</span>\n<span style=\"font-weight: 600; color: #fff\">RESISTENT</span>\n<span style=\"font-weight: 600; color: #fff\">EARBUDS</span>",
+          "content": "<span style=\"font-weight: 600; color: #fff\">SWEAT</span>\n<span style=\"font-weight: 600; color: #fff\">RESISTENT</span>\n<span style=\"font-weight: 600; color: #fff\"> EARBUDS</span>",
```

Comment:
What change was made here? Seems like there might be some invisible chars too.

Reply:
Yea I noticed these in the diff too, but nothing changed here. Just deleted the two circles on page 7 and replace them with the new bullseye shape

Reply:
I see a `U+2028` character (line separator) being added here when checking via https://www.soscisurvey.de/tools/view-chars.php. Maybe it's an OS thing, not sure 🤷 

I remember this caused issues before, see https://github.com/google/web-stories-wp/issues/2473#issuecomment-651509687

Reply:
In 9b0e5b3b234fdf8c84959f365795746cf1c230b8 I've added a quick test for this. Hope that's OK. Reverting the unintended changes here in this file should make it pass.

Reply:
yea absolutely ok. sweet. thanks.

Reply:
Cool. In the future this is could be moved to a linter, bit for now it should work.